src/test/java/org/pesmypetcare/webservice/controller/usermanager/UserRestControllerTest.java

```diff
@@ -0,0 +1,84 @@
+package org.pesmypetcare.webservice.controller.usermanager;
+
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+import org.pesmypetcare.webservice.entity.UserEntity;
+import org.pesmypetcare.webservice.service.UserService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.BDDMockito.willDoNothing;
+import static org.mockito.BDDMockito.willReturn;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.delete;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.get;
+import static org.springframework.test.web.servlet.request.MockMvcRequestBuilders.put;
+import static org.springframework.test.web.servlet.result.MockMvcResultMatchers.status;
+
+@SpringBootTest
+@AutoConfigureMockMvc
+@ExtendWith(MockitoExtension.class)
+class UserRestControllerTest {
+    private static String urlBase;
+    private static String username;
+    private static UserEntity userEntity;
+    private static String jsonEmail;
+    private static String jsonPassword;
+
+    @Autowired
+    private MockMvc mockMvc;
+
+    @MockBean
+    private UserService service;
+
+    @BeforeAll
+    public static void setUp() {
+        urlBase = "/users";
+        username = "PotatoMaster";
+        userEntity = new UserEntity();
+        jsonEmail = "{\n"
+            + "  \"newEmail\": \"xavi@nanako.com\"\n"
+            + "} ";
+        jsonPassword = "{\n"
+            + "  \"newPasswordl\": \"hiufdahs\"\n"
+            + "} ";
+    }
+
+    @Test
+    public void deleteAccountShouldReturnStatusOk() throws Exception {
+        willDoNothing().given(service).deleteById(anyString());
+        mockMvc.perform(delete(urlBase + "/" + username + "/delete"))
+            .andExpect(status().isOk());
+    }
+
+    @Test
+    public void getUserDataShouldReturnUserDataAndStatusOk() throws Exception {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Throwing 'Exception' is not allowed.](https://app.codacy.com/gh/Grupo13-PES-Mascotas/PES_My-Pet-Care-Webservice/pullRequest?prid=5247081)